src/view/dialog/Dialog.tsx

```diff
@@ -35,13 +38,27 @@ export function Dialog() {
     }, [dialogId]);
 
     useEffect(() => {
-        if (!lest) {
+        const listener = () => setActiveTab(!document.hidden);
+        document.addEventListener('visibilitychange', listener, false);
+        return () => document.removeEventListener('visibilitychange', listener);
+    }, [setActiveTab]);
+
+    useEventListener<{ tabId: string }>('veilarbpersonflatefs.tab-clicked', (event) => {
+        if (event.detail.tabId === 'DIALOG') {
+            setActivePersonflateTab(true);
+        } else {
+            setActivePersonflateTab(false);
+        }
+    });
+
+    useEffect(() => {
+        if (!lest && activeTab && activePersonflateTab) {
```

Comment:
Hvorfor både sjekk på document.hidden og aktiv tab i veilarbpersonflate?

Reply:
fordi man kan enten ha en aktiv tab, eller en personflate tab, så dette funker både for brukere og veiledere nå :)